utils/sunwait/Makefile

```diff
@@ -31,16 +31,6 @@ define Package/sunwait/description
 	Has features that make it useful for home automation tasks.
 endef
 
-define Build/Compile
-	$(TARGET_CC) $(TARGET_CPPFLAGS) -o $(PKG_BUILD_DIR)/sunwait.o -c $(PKG_BUILD_DIR)/sunwait.c
-	$(TARGET_CC) $(TARGET_CPPFLAGS) -o $(PKG_BUILD_DIR)/sunriset.o -c $(PKG_BUILD_DIR)/sunriset.c
-	$(TARGET_CC) $(TARGET_CPPFLAGS) -o $(PKG_BUILD_DIR)/print.o -c $(PKG_BUILD_DIR)/print.c
-	$(TARGET_CC) $(TARGET_LDFLAGS) -o $(PKG_BUILD_DIR)/$1 $(PKG_BUILD_DIR)/sunwait.o $(PKG_BUILD_DIR)/sunriset.o $(PKG_BUILD_DIR)/print.o -lm
-endef
```

Comment:
We can keep it, if you want, but we need to add TARGET_CFLAGS:

```
define Build/Compile
        $(TARGET_CC) $(TARGET_CPPFLAGS) $(TARGET_CFLAGS) -c $(PKG_BUILD_DIR)/sunwait.c -o $(PKG_BUILD_DIR)/sunwait.o
        $(TARGET_CC) $(TARGET_CPPFLAGS) $(TARGET_CFLAGS) -c $(PKG_BUILD_DIR)/sunriset.c -o $(PKG_BUILD_DIR)/sunriset.o
        $(TARGET_CC) $(TARGET_CPPFLAGS) $(TARGET_CFLAGS) -c $(PKG_BUILD_DIR)/print.c -o $(PKG_BUILD_DIR)/print.o
        $(TARGET_CC) $(TARGET_LDFLAGS) -o $(PKG_BUILD_DIR)/sunwait \
                $(PKG_BUILD_DIR)/sunwait.o $(PKG_BUILD_DIR)/sunriset.o $(PKG_BUILD_DIR)/print.o -lm
endef 
```

Like this. I was able to compile it.

Reply:
@neheb any opinion on this one?

Reply:
lm is built into libc nowadays. No need for an extra link.

Reply:
Good to know, but my question was rather - remove the entirely section as proposed or keep it explicit like it was, but add TARGET_CFLAGS.

Reply:
Remove.